src/reason_pprint_ast.ml

```diff
@@ -630,9 +630,6 @@ let rules = [
     (TokenPrecedence, (fun s -> (Right, s = "asr")));
   ];
   [
-    (TokenPrecedence, (fun s -> (Left, s.[0] == '#')));
```

Comment:
It should match the order that tokens are specified in the `.mly` file. I believe that means it should match the ranking of `SHARPOP` in the list of tokens. I believe that puts it higher than `isSimplePrefixToken` (which would be things like a leading `BANG` `!`).